doc/conf.py

```diff
@@ -21,28 +35,59 @@
     "replacements",
     "smartquotes",
     "tasklist",
+    "colon_fence",
 ]
 
-exclude_patterns = [
-    "README.md",
-    "_build",
-]
-source_suffix = {
-    ".md": "markdown",
-}
+myst_dmath_double_inline = True
+myst_heading_anchors = 3
+
+# -----------------------------------------------------------------------------
+# BibTeX
+# -----------------------------------------------------------------------------
 
-# https://pypi.org/project/sphinxcontrib-bibtex/
 bibtex_bibfiles = ["references.bib"]
 bibtex_default_style = "unsrt"
 
-myst_dmath_double_inline = True
-myst_heading_anchors = 3
+# -----------------------------------------------------------------------------
+# Sphinx-book-theme
+# -----------------------------------------------------------------------------
 
 html_theme = "sphinx_book_theme"
 html_title = f"Spglib v{release}"
 html_theme_options = {
     # https://sphinx-book-theme.readthedocs.io/en/latest/reference.html
+    "repository_url": "https://github.com/spglib/spglib",
+    "use_repository_button": True,
     "navigation_with_keys": True,
     "globaltoc_includehidden": "true",
+    "show_toc_level": 3,
 }
 html_static_path = ["_static"]
+
+# -----------------------------------------------------------------------------
+# Autodoc2
+# -----------------------------------------------------------------------------
+autodoc2_output_dir = "python-interface/python-apidocs"
+autodoc2_render_plugin = "myst"
+autodoc2_packages = [
+    {
+        "path": "../python/spglib/spglib.py",
+        "module": "spglib",
+    }
+]
+autodoc2_docstring_parser_regexes = [
+    (r".*", 'rst'),
+]
+autodoc2_hidden_objects = ['undoc', 'dunder', 'private', 'inherited']
```

Comment:
Without `private`, private functions like `_expand_cell` are displayed. I activate all flags to show only selected ones.

Reply:
Was thinking these might still be useful for `develop.md`, especially if we later use linkage to-and-from C api, which the IDE cannot handle yet.

Reply:
Then, it will be a balance between the usefulness and the effort to maintain the private functions in `development/develop.md`. I am not so attracted to tabulate these private functions so far.

Reply:
For class members it would be tricky indeed since there would be duplications. Would probably need to look at the actual cases if it makes sense to include them somewhere. For now though, this list is ok.